browser/pages/location/location.html

```diff
@@ -23,21 +23,29 @@
                  ng-init="locCtrl.checkFolder()" ng-change="locCtrl.checkFolder()" ng-model="locCtrl.folder" path-validator autofocus/>
           <button id="location-browse-btn" type="button" class="btn btn-primary location-browse-btn" ng-click="locCtrl.selectFolder()">Browse</button>
 
-          <div ng-show="!locationForm.folder.$invalid && locCtrl.folderExists" class="has-warning">
+          <div ng-show="!locationForm.folder.$invalid && locCtrl.folderExists && !locCtrl.nonasciichars" class="has-warning">
             <strong id="existingFolderStatus" class="help-block">
               <span class="pficon pficon-warning-triangle-o"></span>
               The selected folder already exists, some files might be overwritten.
             </strong>
           </div>
 
-          <div ng-show="!locationForm.folder.$invalid && !locCtrl.folderExists">
+          <div ng-show="!locationForm.folder.$invalid && !locCtrl.folderExists && !locCtrl.nonasciichars">
             <strong id="createFolderStatus" class="help-block">
               <span class="pficon pficon-info"></span>
               The selected folder will be created.
             </strong>
           </div>
 
-          <div ng-show="locationForm.folder.$error" class="help-block has-error">
+          <div ng-show="locCtrl.checkUserProfilestatus() && locCtrl.nonasciichars" class="help-block has-error">
+            <strong id="checkUserProfilestatus" class="help-block has-error">
+              <span class="pficon-error-circle-o"></span>
+              Path to user home directory has Non-ASCII characters
```

Comment:
Another alternative is to move this verification to confirmation page and do validation there and show error message only if Virtualbox is going to be installed. The message can suggest to user manually set TMP in system preferences and restart installer. That validation would work only in case of virtual box installation.